site/blog/2022/deploy-static-website.mdx

```diff
@@ -0,0 +1,193 @@
+---
+slug: deploy-static-website
+title: Deploy a Static Website Using SQL
+date: 2022-12-21
+unlisted: true
+authors: [mtp1376]
+---
+Did you know you can deploy a static website without leaving your SQL REPL? In this post we'll show you how to leverage IaSQL's capabilities to simply deploy a static website from your GitHub repository to AWS using only SQL commands.
```

Comment:
i'd remark the "on AWS" on the title and the first sentence, rather than specifying that later on the second sentence "from your GitHub repository to AWS"

Reply:
+1